### ISSUE TITLE
Ctrl-a s - to send magic strings to hosts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [brockgr/csshx#9]
```
For example:

Ctrl-a s h  - To send hostname (as passed to ssh) to enabled remote machines.

```

Original issue reported on code.google.com by `gavin.brock` on 21 Apr 2009 at 8:33

**Comment by GoogleCodeExporter** [comment]
Original comment by `gavin.brock` on 22 Apr 2009 at 2:55
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Released in 0.61
```

Original comment by `gavin.brock` on 30 Apr 2009 at 2:43
- Changed state: **Done**
